packages/react-core/src/next/components/Modal/ModalBoxHeader.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
```

Comment:
Just a general comment for this component. Do we want to continue supporting a composable and prop based implementation? Or would we want to expose other components like ModalBoxTitle and leave it entirely up to the consumer to pass in those (or whatever) as children to ModalBoxHeader?

Reply:
My concern with that is that it will be easier for consumer to construct the modal incorrectly (e.g. leave out the Header wrapper). Especially with inconstancy construction across components. The Card component has a separate Tile component and a separate Header component (Tilte is not wrapped by header).